Production/crab/configs/Phase2_110X/QCD.txt

```diff
@@ -0,0 +1,8 @@
+Automatic
+globalTag=110X_mcRun4_realistic_v3 sampleType=MC_Phase2_110X
+
+QCD_Pt-15to3000_NoPU         1000 /QCD_Pt-15to3000_TuneCP5_Flat_14TeV-pythia8/Phase2HLTTDRWinter20RECOMiniAOD-NoPU_castor_all_pt_tracks_110X_mcRun4_realistic_v3-v2/MINIAODSIM
```

Comment:
Are the any QCD samples with the correct tau spectrum available (in addition to the flat ones listed here)? The correct spectrum will be needed to reweight tau candidates when computing the ROC curves.

Reply:
What was the name syntax for Run2 samples with correct tau spectrum? In any case, the samples I've added above are the only ones available at the moment, see e.g. [here](https://cmsweb.cern.ch/das/request?view=list&limit=50&instance=prod%2Fglobal&input=dataset+status%3D*+dataset%3D%2FQCD*%2F*Run4*%2FMINIAODSIM)

Reply:
For Run 2 there are pt-binned QCD samples, which are not available for Phase 2. However, maybe it is possible to use MinBias [samples](https://cmsweb.cern.ch/das/request?view=list&limit=50&instance=prod%2Fglobal&input=dataset+dataset%3D%2FMinBias*%2F*Run4*%2FMINIAODSIM) that should have a more physical spectrum.

Reply:
Done